src/components/calculator/PrevalenceControls.tsx

```diff
@@ -231,124 +231,118 @@ export const PrevalenceControls: React.FunctionComponent<{
         </div>
       )}
 
-      <span
-        className={
-          'expandable-header ' + (isManualEntryCurrently ? 'd-none' : '')
-        }
-        onClick={() => setDetailsOpen(!detailsOpen)}
-        aria-controls="prevelance-details"
-        aria-expanded={detailsOpen}
+      <Expandable
+        id="prevelance-details"
+        header="Details"
+        headerClassName={isManualEntryCurrently ? 'd-none' : ''}
+        defaultState={detailsOpen}
+        setter={setDetailsOpen}
```

Comment:
This doesn't work. If you click "manual entry" it no longer opens up the details, since setDetailsOpen has no way to propagate that to Expandable.

Instead, I think you want to break Expandable into two components:

ControlledExpandable has no internal state, and have `setter` and `isOpen` as mandatory variables.

Expandable wraps ControlledExpandable with  useState. It takes neither `setter` nor `isOpen`, but can take `defaultState`

In general, if there are two completely different ways a component can be managed, its probably a sign that it should be two different components.

Reply:
Thanks.  I knew there was a better way as I was writing it. I'll do it as you suggested!

Reply:
Done!